src/main/scala/xiangshan/backend/fu/PMA.scala

```diff
@@ -110,7 +110,9 @@ trait PMAMethod extends PMAConst {
       MemMap("h00_3801_0000", "h00_3801_FFFF",   "h0", "BEU",         "RW"),
       MemMap("h00_3802_0000", "h00_3802_0FFF",   "h0", "DebugModule", "RWX"),
       MemMap("h00_3802_1000", "h00_3802_1FFF",   "h0", "MMPMA",       "RW"),
-      MemMap("h00_3802_2000", "h00_3900_0000",   "h0", "Reserved",    ""),
+      MemMap("h00_3802_2000", "h00_3802_207F",   "h0", "L1DCacheCtrl", "RW"), // PR#4009
```

Comment:
NOTE: Conflict with #4009, address space `0x38022000~0x3802207F` is reserved for that PR